presentation/src/main/java/com/example/dms_android/feature/auth/login/SignInViewEffect.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.dms_android.feature.auth.login
+
+sealed class SignInViewEffect {
+    object SignInSuccess : SignInEvent()
```

Comment:
event는 동작을 의미하는게 좋아보여요
다른 것도 부탁드립니다
```suggestion
    object NavigateToHome : SignInEvent()
```

Reply:
넵 알겠습니다